ci/test/lint-main/checks/check-python-discouraged.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# check-python-discouraged.sh — check discouraged commands.
+
+set -euo pipefail
+
+cd "$(dirname "$0")/../../../.."
+
+. misc/shlib/shlib.bash
+
+if [[ ! "${MZDEV_NO_PYTHON:-}" ]]; then
+
+  SINGLE_LINE_MATCHES=$(awk '/\.run\("testdrive",/ {print FILENAME ":" FNR} {prev_line = $0}' misc/python/materialize/**/*.py test/**/*.py)
+  MULTI_LINE_MATCHES=$(awk '/"testdrive",/ && prev_line ~ /\.run\($/ {print FILENAME ":" FNR} {prev_line = $0}' misc/python/materialize/**/*.py test/**/*.py)
+  # TODO
+  # MULTI_LINE_MATCHES=$(echo "$MULTI_LINE_MATCHES" | grep -v "misc/python/materialize/mzcompose/composition.py")
+  MATCHES="${SINGLE_LINE_MATCHES}
+${MULTI_LINE_MATCHES}"
+
+  if [[ "$MATCHES" != "\n" ]]; then
+      echo "Use .run_testdrive( instead of .run(\"testdrive\":"
+      echo "$MATCHES"
+      try exit 1
```

Comment:
Don't we have a `fail` or something similar? Couldn't find anything like that.

Reply:
Did you mean to suggest `c.testdrive(` here? We do not seem to have `run_testdrive(`.

Reply:
assert False?